src/main/webapp/app/file-upload-assessment/file-upload-assessment.component.html

```diff
@@ -0,0 +1,75 @@
+<jhi-assessment-layout
+    (navigateBack)="goToExerciseDashboard()"
+    [isLoading]="isLoading"
+    [busy]="busy"
+    [isAssessor]="isAssessor"
+    [isAtLeastInstructor]="isAtLeastInstructor"
+    [canOverride]="canOverride"
+    [result]="result"
+    [assessmentsAreValid]="assessmentsAreValid"
+    [complaint]="complaint"
+    (save)="onSaveAssessment()"
+    (submit)="onSubmitAssessment()"
+    (cancel)="onCancelAssessment()"
+    (nextSubmission)="assessNextOptimal()"
+    (updateAssessmentAfterComplaint)="onUpdateAssessmentAfterComplaint($event)"
+>
+    <div class="row mt-4 resizable-horizontal" *ngIf="!busy && exercise" [ngStyle]="{ 'min-height.px': 500 }">
+        <div class="row flex-nowrap col-12">
+            <div class="card flex-grow-1 mr-4 file-upload-assessment">
+                <div class="card-header bg-primary text-white">
+                    <span *ngIf="exercise.title">{{ exercise.title }}</span>
+                    <span class="flex-fill"></span>
+                    <jhi-score-display [score]="totalScore" [maxScore]="exercise.maxScore"></jhi-score-display>
+                </div>
+
+                <div class="card-body">
+                    <div *ngIf="submission.filePath" class="card-text">
+                        <a class="text-primary" (click)="downloadFile(submission.filePath)">{{ 'artemisApp.fileUploadAssessment.submissionFile' | translate }}</a>
+                        <span class="ml-1 badge badge-info">
+                            {{ attachmentExtension(submission.filePath!) | uppercase }}
+                        </span>
+                    </div>
+                    <button class="btn btn-success" (click)="addReferencedFeedback()">{{ 'artemisApp.fileUploadAssessment.addFeedback' | translate }}</button>
+                </div>
+            </div>
+
+            <jhi-resizable-instructions
+                class="resizable-submission card"
+                id="instructions"
+                [ngStyle]="{ 'width.px': 400, 'min-width.px': 400 }"
```

Comment:
I would prefer using a component scss file here, the styling can be hard to find when issues come up :see_no_evil:

Reply:
You are right!